cloudfront/iiif/edge-lambda/src/test_event_request.ts

```diff
@@ -0,0 +1,28 @@
+import { CloudFrontRequestEvent } from 'aws-lambda';
+
+const request = (
+    uri: string,
+    querystring?: string
+): CloudFrontRequestEvent => ({
+  Records: [
+    {
+      cf: {
+        config: {
+          distributionId: 'EXAMPLE',
+          distributionDomainName: '',
+          requestId: '',
+          eventType: 'origin-request',
+        },
+        request: {
+          uri,
+          querystring: querystring || '',
+          method: 'GET',
+          clientIp: '2001:cdba::3257:9652',
+          headers: {},
+        },
+      },
+    },
+  ],
```

Comment:
I assume this is from https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-event-structure.html#example-origin-request; having a link in the code to the event this is based on would be helpful.

Reply:
It's cargo-culted from the wellcomecollection.org repo - though it looks to have been sourced from there originally. Will add comment.